### ISSUE TITLE
Blocked domain

### ISSUE THREAD

**Issue by N0cky** [hagezi/dns-blocklists#4053]
### Which AdBlocker/DNS cloud service do you use?

AdGuard Home

### ControlD users

- [ ] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [ ] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Threat Intelligence Feeds

### Which domain(s) should be unblocked?

bunk.at

### Why should the domain(s) be unblocked?

The domain bunk.at is my personal domain, and its inclusion in the blocklist is causing unintended restrictions on my own services.

### Privacy

- [X] I confirm that the report does not contain any private information.

**Comment by hagezi** [comment]
@N0cky via https://www.stopforumspam.com/downloads/toxic_domains_whole.txt

**Comment by hagezi** [comment]
@N0cky This domain is also flagged by some vendors: https://www.virustotal.com/gui/domain/bunk.at/detection

**Comment by N0cky** [comment]
Thank you! There's probably not much I can do about VirusTotal, but I will try to get my domain removed from StopForumSpam.

**Comment by hagezi** [comment]
@N0cky Yes, having the flags removed is stupid work, you will have to contact every vendor. As long as the flags exist, the domain will probably end up on a list from time to time.

I have checked the most common secure DNS. The domain is only blocked by the public CleanBrowsing DNS.
